### PR TITLE
🚀 [Story performance] Removed spaces in strings

### PR DESCRIPTION
**Pull request by mszylkowski**
Some parts of the code used to have extra spaces due to splitting strings into multiple newlines, making the indentation part of the bundle size.

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story-embedded-component.js [hunk 1]
```diff
@@ -229,13 +229,6 @@ export class AmpStoryEmbeddedComponent {
         this.setState_(EmbeddedComponentState.HIDDEN, null /** component */);
         break;
       case EmbeddedComponentState.FOCUSED:
-        if (this.state_ !== EmbeddedComponentState.HIDDEN) {
```
Comment (kristoferbaxter) [inline extensions/amp-story/1.0/amp-story-embedded-component.js:232]: Intentional?

Comment (mszylkowski) [reply extensions/amp-story/1.0/amp-story-embedded-component.js:232]: Yes, we used to have 3 states (these ones + `EXPANDED`) but now we just have 2, so this check is no longer necessary.

It used to warn when we went from `EXPANDED` to `FOCUSED` which didn't make sense.

Comment (kristoferbaxter) [reply extensions/amp-story/1.0/amp-story-embedded-component.js:232]: Got it, opportunistic cleanup.

Thanks!

**Review by kristoferbaxter — APPROVED** [review]

**Comment by gmajoulet** [comment]
This PR increased the bundle size (`+0.15KB`), why was this needed?

**Pull request merged**